### ISSUE TITLE
Messy graph output

### ISSUE THREAD

**Issue by iD4rksid3** [forensicxlab/VISION-ProcMon#5]
OS: Windows 11
Version: 1.4

When applying the (File Operation) on the attached CSV file the output graph gets messy as in the following GIF:

![test](https://github.com/forensicxlab/VISION-ProcMon/assets/31139699/2d8e0c94-7137-4b9e-9792-5ced4305d31d)

and here is the example CSV file:
[example.csv](https://github.com/forensicxlab/VISION-ProcMon/files/12169612/example.csv)

**Comment by forensicxlab** [comment]
Hello, 
Thank you for testing the tool and declaring the issue. 
I will look into this, should be fixed in the next release.


Best regards.

**Comment by YoavLevi** [comment]
> 
Same problem for me.

**Comment by forensicxlab** [comment]
Dear all, 

The issue is now fixed on dev and the update should be available soon.
I have also noticed that with small screen, the left panel was not so good.
You will be able to organize the menu and see more details in the next release. 

Example bellow using the file provided for the issue:

![example](https://github.com/forensicxlab/VISION-ProcMon/assets/100540050/a271dfad-283e-4d8e-8fcb-8e5901514211)

Also working on a "step count" cursor to move in time.
I will close this issue once the update is available.
Best regards.
